app/components/candidate_interface/new_reference_history_component.rb

```diff
@@ -21,5 +21,9 @@ def formatted_title(event)
         event.name.humanize
       end
     end
+
+    def can_be_cancelled?
+      reference.feedback_status == 'feedback_requested'
```

Comment:
Oh just this condition? But is this show in all histories of the reference? I think the cancel link should only show on the request reference event. I think there may be a bug. Unless I understood wrong. A way to make it work is something like this:

```ruby
  def can_be_cancelled?(event)
    reference.feedback_requested? && event.name == 'request_sent'
  end
```

Can you test when you have many histories into one reference (send many reminders for example on a request reference).

Reply:
The `reference.feedback_status` is essentially the last event so I don't think we need to pass it in. Sending reminders doesn't change `feedback_status` so there should be no bug there, I've added a test to cover that scenario.

Reply:
Not sure how this helps, sending reminders doesn't change the feedback_status. I've added a test to cover, happy to discuss

Reply:
Sorry, I mean the bug is that using only the feedback status as condition does it mean that the cancel request link will appear in each event of the reference.

Reply:
but surely feedback status can only be 'feedback_requested' if the 'request_sent' event has occurred?